_includes/head.html

```diff
@@ -2,6 +2,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1"> 
+    <meta name="google-site-verification" content="NGcFSsc8cnsbnE_lbtEHYoWWSNe-o8S6S_ZhsFr5htI" />
```

Comment:
`meta` elements are void elements, they don't allow self-closing start tags nor end tags.

Reply:
Oops :) I'll update that now

Reply:
Interestingly, that was the code Google provided me with...

Reply:
It'd be necessary for XHTML. I forget what user agents are supposed to do with self-closed HTML elements (as opposed to foreign) but I suspect it's just to have a warning (that no browser would ever show to users).